### ISSUE TITLE
label.py is missed

### ISSUE THREAD

**Issue by lizhuorong** [phillipi/pix2pix#106]
Hi, thanks for the awesome implementation !

The label.py which is mentioned in line 81 of cityscapes.py cannot be found and consequently an error that "No module named labels" raises when evaluating.  

Besides, I feel confused that whether I can successfully conduct the label2photo evaluation on cityscapes  as I utilized the tensorflow implementation of your project (https://github.com/affinelayer/pix2pix-tensorflow) rather then the torch version? What else needs to be done besides what you have suggested?

I deeply appreciate your sharing codes and look forward to your reply.

**Comment by tinghuiz** [comment]
`labels.py' is from the original Cityscapes toolkit (https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py) and imported by this line: https://github.com/phillipi/pix2pix/blob/master/scripts/eval_cityscapes/cityscapes.py#L18. 

Using the tensorflow implementation should be fine. You just need to save the predictions as specified in the README. You would need to install Caffe to run the evaluation though as FCN model was trained with Caffe.

**Comment by lkiani** [comment]
> `labels.py' is from the original Cityscapes toolkit (https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py) and imported by this line: https://github.com/phillipi/pix2pix/blob/master/scripts/eval_cityscapes/cityscapes.py#L18.
> 
> Using the tensorflow implementation should be fine. You just need to save the predictions as specified in the README. You would need to install Caffe to run the evaluation though as FCN model was trained with Caffe.

I downloaded the labels.py file and placed it on the specified path, but I still receive the No module named labels message.
File path: content / pix2pix / helpers / labels.py
thank you

**Comment by lkiani** [comment]
> `labels.py' is from the original Cityscapes toolkit (https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py) and imported by this line: https://github.com/phillipi/pix2pix/blob/master/scripts/eval_cityscapes/cityscapes.py#L18.
> 
> Using the tensorflow implementation should be fine. You just need to save the predictions as specified in the README. You would need to install Caffe to run the evaluation though as FCN model was trained with Caffe.

I downloaded the labels.py file and placed in content / pix2pix / scripts / helpers / but I still receive the No module named labels message.
thank you
